plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseHiveConnectorTest.java

```diff
@@ -185,10 +185,15 @@ protected BaseHiveConnectorTest()
     protected static QueryRunner createHiveQueryRunner(Map<String, String> extraProperties, Consumer<QueryRunner> additionalSetup)
             throws Exception
     {
+        // Use faster compression codec in tests. TODO remove explicit config when default changes
+        verify(new HiveConfig().getHiveCompressionCodec() == HiveCompressionOption.GZIP);
+        String hiveCompressionCodec = HiveCompressionCodec.ZSTD.name();
```

Comment:
Why not to inline this ?

Reply:
you mean to use "ZSTD" string literal? i think the current way describes which enum this is referring to